src/.pre-commit-config.yaml.jinja

```diff
@@ -235,3 +229,12 @@ repos:
         args:
           - --rcfile=.pylintrc-mandatory
 {%- endif %}
+  - <<: *maintainer_tools
```

Comment:
not sure to understand this syntax.

Reply:
It's reusing the anchor defined above. Check this out: https://www.linode.com/docs/guides/yaml-anchors-aliases-overrides-extensions/